incubator/hnc/pkg/controllers/setup.go

```diff
@@ -43,12 +43,14 @@ func Create(mgr ctrl.Manager, f *forest.Forest, maxReconciles int, newObjectCont
 func createObjectReconciler(newObjectController bool, mgr ctrl.Manager, f *forest.Forest, gvk schema.GroupVersionKind) (NamespaceSyncer, error) {
 	if newObjectController {
 		or := &ObjectReconcilerNew{
-			Client: mgr.GetClient(),
-			Log:    ctrl.Log.WithName("controllers").WithName(gvk.Kind),
-			Forest: f,
-			GVK:    gvk,
+			Client:   mgr.GetClient(),
+			Log:      ctrl.Log.WithName("controllers").WithName(gvk.Kind),
+			Forest:   f,
+			GVK:      gvk,
+			Affected: make(chan event.GenericEvent),
 		}
-		return or, or.SetupWithManager(mgr)
+		// TODO change the hardcoded 10
```

Comment:
Can we expand this comment a bit so someone without full context can understand what that action of the TODO should be? What considerations should be made on figuring out max concurrent reconciles and whether 10 is arbitrary, educated guess, or specific.

Reply:
I created a github issue and linked it there with a comment.